conformance_tests/sysman/test_sysman_init/CMakeLists.txt

```diff
@@ -27,4 +27,19 @@ add_lzt_test(
     src/test_init_sysman_before_core.cpp
     src/main.cpp
     LINK_LIBRARIES level_zero_tests::logging level_zero_tests::utils)
-    
\ No newline at end of file
+
+add_lzt_test(
+    NAME test_init_sysman_from_core_handle
```

Comment:
could we name it different ? it should convey we are doing init and validating one sysman api using core/sysman device handle

Reply:
I can change the name to ```test_init_sysman_for_freq_domains_from_core_handle``` . But won't the file name be too long?

Reply:
yes, we have to think of a small name conveying the meaning. Current name **test_init_sysman_from_core_handle** mis conveys that sysman is initialized using core handle. But actually we are initializing both core and sysman separately and checking one sysman API if it's not crashing.

Reply:
I will keep test_init_sysman_from_core_handle and test_init_sysman_from_sysman_handle for now. The detailed test case name will anyways there in the test case definition. Difficult to have a small file name.